ios/FluentUI.Demo/FluentUI.Demo/Demos.swift

```diff
@@ -13,6 +13,7 @@ let demos: [(title: String, controllerClass: UIViewController.Type)] = [
     ("BadgeField", BadgeFieldDemoController.self),
     ("BadgeView", BadgeViewDemoController.self),
     ("Button", ButtonDemoController.self),
+	("Button (vNext)", ButtonVNextDemoController.self),
```

Comment:
weird tabbing .. also is your Xcode index set to spaces or tabs?

Reply:
swift-lint should catch this?

Reply:
@rdeassis will have to install swiftlint on his machine for it to run as part of the build

Reply:
I haven't ran swift-lint on the files. It should be fixed now...